source/javascripts/application.test.js

```diff
@@ -22,39 +22,39 @@ describe("Application JS", () => {
   }); 
 
   it("should always call the showBannerIfConsentNotSet", () => {
-    require("./application")
+    require("./application.min.js")
```

Comment:
Just wondering if the original tests were still working?
As I was expecting this wouldn't need to be changed.

Reply:
They didn't work for me  locally, but Github doesn't seem to mind whatever I put so I'm guessing they're not being run as part of the static build process?

Reply:
I would probably leave it as `/application`.
So this way, we are only getting your change in the PR.
I suspect that in github actions, it does not have the same issue.

Reply:
Cool. Will do!